packages/terra-paginator/src/Paginator.module.scss

```diff
@@ -25,42 +46,61 @@
       border-radius: var(--terra-paginator-nav-link-border-radius, 3px);
       color: var(--terra-paginator-nav-link-color, #0059a8);
       cursor: pointer;
       display: inline-block;
-      line-height: var(--terra-paginator-nav-link-line-height, 1.43);
+      font-family: inherit;
+      font-size: var(--terra-paginator-nav-link-font-size, 1rem);
+      line-height: var(--terra-paginator-nav-link-line-height, 1.429);
+      margin: 0; /* Reset margin for Safari. Safari defaults to 2px. */
       margin-left: var(--terra-paginator-nav-link-margin-left, 0.14285rem);
       margin-right: var(--terra-paginator-nav-link-margin-right, 0.14285rem);
+      max-width: 100%; /* Need to help with overflow-wrap */
       min-width: var(--terra-paginator-nav-link-min-width, 2rem); // Helps ensure limited resizing as you page through large numbers.
+      outline: none;
+      overflow-wrap: break-word; /* Modern browsers */
       padding: var(--terra-paginator-nav-link-padding, 0.35714rem 0.5rem);
       text-align: center;
-      user-select: none; // Keeps user from highlighting the paging link text.
-
+      text-decoration: var(--terra-paginator-nav-link-text-decoration, none);
+      touch-action: manipulation; // Enable fast tap interaction in webkit browsers; see https://webkit.org/blog/5610/more-responsive-tapping-on-ios/
+      user-select: none; // Keeps user from highlighting and text selection
+      white-space: normal;
+      word-wrap: break-word; /* For IE 10 and IE 11 */
+  
       &:hover {
-        background: var(--terra-paginator-nav-link-hover-background);
+        background: var(--terra-paginator-nav-link-hover-background, none);
+        border: var(--terra-paginator-nav-link-hover-border, none);
         color: var(--terra-paginator-nav-link-hover-color, #001f67);
         text-decoration: var(--terra-paginator-nav-link-hover-text-decoration, underline);
       }
 
+      // Remove the inner border and padding in Firefox.
+      &::-moz-focus-inner {
+        border: 0;
+        padding: 0;
+      }
+  
       &:focus {
-        background: var(--terra-paginator-nav-link-focus-background);
-        border: var(--terra-paginator-nav-link-focus-border);
+        background: var(--terra-paginator-nav-link-focus-background, none);
+        border: var(--terra-paginator-nav-link-focus-border, none);
         box-shadow: var(--terra-paginator-nav-link-focus-box-shadow, 0 0 0 2px rgba(76, 178, 233, 0.25), inset 0 0 0 1px #4cb2e9);
-        color: var(--terra-paginator-nav-link-focus-color);
+        color: var(--terra-paginator-nav-link-focus-color, #0059a8);
         outline: none;
       }
-
+  
       &:active {
         background: var(--terra-paginator-nav-link-active-background, rgba(0, 0, 0, 0.07));
+        border: var(--terra-paginator-nav-link-active-border, none);
         color: var(--terra-paginator-nav-link-active-color, #001f67);
+        -webkit-tap-highlight-color: transparent;
```

Comment:
Was this testes with a border color in firefox? We likely need to make some of these changes since firefox doesn't trigger onFocus or onBlur for buttons, so in terra-button we had to be managed in state for consistent styling across browsers: 
https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L131
https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L143
https://github.com/cerner/terra-core/blob/master/packages/terra-button/src/Button.jsx#L195

Reply:
Thanks @emilyrohrbough - will have to test it. Managing the button focus-glow after clicking was still an outstanding item this PR didn't account for. Will see what needs to be added while waiting for translations.

Reply:
Will leave #1516 for followup enhancement.